src/ui.cpp

```diff
@@ -534,23 +538,31 @@ UpdateDialog::UpdateDialog()
     m_buttonSizer = new wxBoxSizer(wxHORIZONTAL);
 
     m_updateButtonsSizer = new wxBoxSizer(wxHORIZONTAL);
-    m_updateButtonsSizer->Add
-                          (
+
+
+    m_SkipVersionButtonSizer = new wxBoxSizer(wxHORIZONTAL);
+
+    m_SkipVersionButtonSizer->Add(
                             new wxButton(this, ID_SKIP_VERSION, _("Skip this version")),
-                            wxSizerFlags().Border(wxRIGHT, PX(20))
-                          );
-    m_updateButtonsSizer->AddStretchSpacer(1);
-    m_updateButtonsSizer->Add
-                          (
+                            wxSizerFlags()
+    );
+
+    m_RemindLaterButtonSizer = new wxBoxSizer(wxHORIZONTAL);
+    m_RemindLaterButtonSizer->Add(
                             new wxButton(this, ID_REMIND_LATER, _("Remind me later")),
-                            wxSizerFlags().Border(wxRIGHT, PX(10))
+                            wxSizerFlags()
                           );
+
+    m_buttonSizer->Add(m_SkipVersionButtonSizer, wxSizerFlags(0));
+    m_buttonSizer->AddStretchSpacer(1);
+    m_buttonSizer->Add(m_RemindLaterButtonSizer, wxSizerFlags(0));
+
     m_updateButtonsSizer->Add
                           (
                             m_installButton = new wxButton(this, ID_INSTALL, _("Install update")),
                             wxSizerFlags()
                           );
-    m_buttonSizer->Add(m_updateButtonsSizer, wxSizerFlags(1));
+    m_buttonSizer->Add(m_updateButtonsSizer, wxSizerFlags(0));
```

Comment:
I don't understand what's this for. If it is needed, then you can remove the entire ", wxSizerFlags(0)" part (that's the default) and the `AddStretchSpacer()` call above (there's nothing to stretch).